dev-docs/pb10-notes.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page_v2
+title: Prebid.js 10.0 Release Notes & Publisher API Changes
+description: Description of the breaking changes included for Prebid.js 10.0
+sidebarType: 1
+---
+
+# Prebid.js 10.0 Bidder Interface and Publisher API Changes
+
+{:.no_toc}
+
+This document describes the changes included for Prebid.js version 10.0.
+
+* TOC
+{:toc}
+
+## Publisher Summary
+
+1. A large number of obsolete modules have been removed. Many modules have changed name. See below for the list.
+2. The legacy method of native targeting keys, `sendTargetingKeys`, has been removed.
+3. `pbadslot` has been removed from the preAuction module. Use `ortb2Imp.ext.gpid` instead.
+4. The API methods `getBidResponses` and `getNoBidsForAdUnitCode` now return arrays of bids.
+5. TypeScript support has landed and Node.js 20+ is required to build.
```

Comment:
6. Using prebid as an NPM dependency no longer requires using Babel and Prebid's Babel configuration.